mlrun/projects/project.py

```diff
@@ -2957,3 +2949,15 @@ def _has_module(handler, kind):
 
 def _is_imported_artifact(artifact):
     return artifact and isinstance(artifact, dict) and "import_from" in artifact
+
+
+def _get_item_absolute_path(url, project):
+    in_context = False
```

Comment:
maybe this should be moved to the utils file 🤔  what is your opinion?

Reply:
this is relevant for projects only.
where will it be reused if not from projects?

Reply:
I agree with Liran, this is why I placed it in the project file rather than the utils file in the first place

Reply:
moved to `MLrunProject` and removed use from `MLrunProjectLegacy`